auth_oauth_multi_token/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'auth_oauth_multi_token',
+    'version': '1.0',
+    'license': 'AGPL-3',

```

Comment:
you include the text of the GPL license, but your module uses AGPL. You can probably drop the license text. If you use the recommended README.rst template for your module, you'll get a link to the license text.